UrologicalSupplies/R4/resources/Questionnaire-R4-UrologicalSuppliesProgressNote.json

```diff
@@ -0,0 +1,1799 @@
+{
+  "resourceType": "Questionnaire",
+  "id": "UrologicalSuppliesProgressNote",
+  "name": "UrologicalSuppliesProgressNote",
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqif-library",
+      "valueReference": {
+        "reference": "Library/UrologicalSupplies-prepopulation"
+      }
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqif-library",
+      "valueReference": {
+        "reference": "Library/BasicPatientInfo-prepopulation"
+      }
+    },
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/cqif-library",
+      "valueReference": {
+        "reference": "Library/BasicPractitionerInfo-prepopulation"
+      }
+    }
+  ],
+  "title": "UrologicalSupplies Progress Note Questionnaire",
+  "status": "draft",
+  "date": "2020-04-09",
+  "publisher": "Da Vinci DTR",
+  "subjectType": ["Patient"],
+  "contained": [
+    {
+      "resourceType": "ValueSet",
+      "id": "gender",
+      "url": "#gender",
+      "status": "draft",
+      "expansion": {
+        "contains": [
+          {
+            "code": "male",
+            "display": "Male"
+          },
+          {
+            "code": "female",
+            "display": "Female"
+          },
+          {
+            "code": "other",
+            "display": "Other"
+          }
+        ]
+      }
+    },
+    {
+      "resourceType": "ValueSet",
+      "id": "yesno",
+      "url": "#yesno",
+      "status": "draft",
+      "expansion": {
+        "contains": [
+          {
+            "code": "Y",
+            "display": "Yes"
+          },
+          {
+            "code": "N",
+            "display": "No"
+          }
+        ]
+      }
+    }
+  ],
+  "item": [
+    {
+      "linkId": "PBD",
```

Comment:
Do we already using this format for the linkId yet?

Reply:
Changed back